### ISSUE TITLE
Show language plugin successfully installed message after telemetry message.

### ISSUE THREAD

**Issue by Apoorva-GA** [getgauge/gauge#1450]
### Expected Behaviour : 
 Show language plugin successfully installed message after telemetry message.

### Steps to reproduce:
- uninstall js plugin 
- open a js project in vscode
- install js plugin in vscode
- see telemetry message over shadows success message

**Comment by Apoorva-GA** [comment]
Fix available in nightly >= 6/7/19

**Comment by Debashis9012** [comment]
Telemetry message appearing two times. Tested with gauge:  ```1.0.6.nightly-2019-07-08```
Kindly find below console log.

```
 jsProjectTest gauge run            

Telemetry
---------

This installation of Gauge collects usage data in order to help us improve your experience.
The data is anonymous and doesn't include command-line arguments.
To turn this message off opt in or out by running 'gauge telemetry on' or 'gauge telemetry off'.

Read more about Gauge telemetry at https://gauge.org/telemetry

(node:45631) DeprecationWarning: grpc.load: Use the @grpc/proto-loader module with grpc.loadPackageDefinition instead
# Getting Started with Gauge
  ## Search Taiko Repository     ✔DEPRECATION WARNING: inputField is deprecated, use textBox api instead
 ✔ ✔

Successfully generated html-report to => /Users/work/taikoProject/issue557/jsProjectTest/reports/html-report/index.html
Specifications: 1 executed      1 passed        0 failed        0 skipped
Scenarios:      1 executed      1 passed        0 failed        0 skipped

Total time taken: 6.936s

Telemetry
---------

This installation of Gauge collects usage data in order to help us improve your experience.
The data is anonymous and doesn't include command-line arguments.
To turn this message off opt in or out by running 'gauge telemetry on' or 'gauge telemetry off'.

Read more about Gauge telemetry at https://gauge.org/telemetry

```

**Comment by Apoorva-GA** [comment]
Fix should be available in gauge-nightly >= 9/7/19

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the tested version: ```version: 1.0.6.nightly-2019-07-09```
